src/backend/distributed/commands/table.c

```diff
@@ -1070,6 +1077,30 @@ AlterTableCommandTypeIsTrigger(AlterTableType alterTableType)
 }
 
 
+/*
+ * DropsForeignKey returns true if the given AlterTableStmt drops a foreign key
+ * for the given relationId. False otherwise.
+ */
+static bool
+DropsForeignKey(Oid relationId, AlterTableStmt *alterTableStatement)
+{
+	if (list_length(alterTableStatement->cmds) != 1)
```

Comment:
Why don't we iterate over the `cms` and only check for == `1`. Today Citus doesn't support multiple subcommands in ALTER TABLE, but let make this code robust to such changes and iterate on the cmds and do the following code per iteration